clients/admin-ui/src/app/store.ts

```diff
@@ -174,7 +177,6 @@ export const makeStore = (preloadedState?: Partial<RootState>) =>
         plusApi.middleware,
         privacyRequestApi.middleware,
         scannerApi.middleware,
-        systemApi.middleware,
```

Comment:
I swapped our system slice to import from `baseApi` so that it could properly invalidate the datamap tag (one small part of https://github.com/ethyca/fides/issues/1386 which we haven't done yet)